### PR TITLE
DIV-5475: Adding new event to issue AOS pack (offline) for Co-Respondent

### PR DESCRIPTION
**Pull request by iichr**
Adding new event to issue AOS pack (offline) for the co-respondent 
https://tools.hmcts.net/jira/browse/DIV-5475

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEvent.json [hunk 1]
```diff
@@ -2197,5 +2197,19 @@
...
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_CoRespondent",
+    "Name": "Issue AOS offline - Co-Respondent",
+    "Description": "Issue AOS (offline) for Co-Respondent",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
```
Comment (ravinderpal) [inline definitions/divorce/json/CaseEvent.json]: By doing this we will have this event showing up at every stage of the application. Are we sure we want this? Is this how is should be?

Comment (ravinderpal) [reply definitions/divorce/json/CaseEvent.json]: Also, you'll see this option regardless of the reason for divorce being Adultery or not

Comment (jvictorcf) [reply definitions/divorce/json/CaseEvent.json]: Good points. Did we talk about a few specific pre-conditions last week @iichr ? Does your story say something about that?

Comment (iichr) [reply definitions/divorce/json/CaseEvent.json]: This is all that's in the story: 
> Pre-condition and post-condition states are no longer relevant for the co-respondent.
So not quite clear, open to interpretation

Comment (iichr) [reply definitions/divorce/json/CaseEvent.json]: Amended per the discussion last week, pre-conditions can be aosStarted and aosAwaiting

File: definitions/divorce/json/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -4772,5 +4772,40 @@
     "CaseEventID": "IssueAOSOffline_Respondent",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "IssueAOSOffline_CoRespondent",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
```
Comment (ravinderpal) [inline definitions/divorce/json/AuthorisationCaseEvent.json]: @iichr @jvictorcf 
Shouldn't this be 'CRU' ? Otherwise no one can trigger the event

**Pull request merged**